editor/src/components/canvas/canvas-strategies/reparent-strategy-helpers.ts

```diff
@@ -148,3 +163,104 @@ export function getReparentTargetForFlexElement(
     }
   }
 }
+
+const propertiesToRemove: Array<PropertyPath> = [
```

Comment:
Now this isn't in a specific strategy, it could probably do with a better name.

Reply:
fixed!